src/main/java/com/example/sinitto/member/controller/MemberControllerAdvice.java

```diff
@@ -0,0 +1,24 @@
+package com.example.sinitto.member.controller;
+
+import com.example.sinitto.member.exception.MemberNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ProblemDetail;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.net.URI;
+
+@RestControllerAdvice
```

Comment:
```suggestion
@RestControllerAdvice(basePackages = "com.example.sinitto.member")
````
예외처리 범위를 설정하는게 어떨까요?

Reply:
적용했습니다 :)